db/migrate/20221122214323_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly change user to author according to the project requirement so it will create author_id with each index
` t.references :author, foreign_key: { to_table: 'users' }`

Reply:
Fixed, thank you! 👍🏼 @VICTOR-CHUKWU